src/components/License/License.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="license text-dark-charcoal-70">
+    <span class="icons text-lg">
+      <i v-if="isCC" class="icon mr-1 cc-logo" title="CC" /><i
+        v-for="(name, index) in icons"
+        :key="index"
+        class="icon mr-px"
+        :class="`cc-${name}`"
+        :title="name.toUpperCase()"
+      />
+    </span>
+    <span class="name">
+      {{ $t(`license-names.${license}`) }}
+    </span>
+  </div>
+</template>
+
+<script>
+import { computed } from '@nuxtjs/composition-api'
+
+/**
+ * Displays the icons for the license along with a readable display name for the
+ * license.
+ */
+export default {
+  name: 'License',
+  props: {
+    /**
+     * the slug of the license
+     * @values
+     */
+    license: {
+      type: String,
+      required: true,
+      validator: (val) =>
+        [
+          'by',
+          'by-sa',
+          'by-nd',
+          'by-nc',
+          'by-nc-sa',
+          'by-nc-nd',
+          'cc0',
+          'pdm',
```

Comment:
surprised this isn't a constant yet somewhere, but we can add that to the frontend later if needed

Reply:
I had a lot of license-related code in the chooser: https://github.com/creativecommons/chooser/blob/main/src/utils/licenses.js :)

Reply:
Extracted constants separately in a67800c.